### PR TITLE
Error Message Hints at Allowed Version

### PR DESCRIPTION
**Pull request by philippfromme**
* incorporates https://github.com/camunda/bpmnlint-plugin-camunda-compat/pull/50 and hints at the allowed version if it exists

image

---

Related to https://github.com/camunda/bpmnlint-plugin-camunda-compat/pull/50

### CODE REVIEW

File: lib/utils/error-messages.js [hunk 1]
```diff
@@ -339,72 +343,57 @@ function getPropertyRequiredErrorMessage(report, executionPlatformLabel) {
...
-function secondLast(array) {
-  return array[array.length - 2];
-}
+function getSupportedMessage(prefix, executionPlatformVersion, allowedVersion) {
+  if (allowedVersion) {
+    return `${ prefix } is only supported by ${ getExecutionPlatformLabel('Camunda Cloud', allowedVersion) } or newer`;
+  }
+
+  return `${ prefix } is not supported by ${ getExecutionPlatformLabel('Camunda Cloud', executionPlatformVersion) }`;
+}
```
Comment (marstamm) [inline lib/utils/error-messages.js:399]: We are hard-coding the Execution platform to `Camunda Cloud` here. This assumes that this error type/utility will never be used for Camunda 7. 
Is there a reason we are not using the platform from the XML?https://github.dev/camunda/linting/blob/8a26cbef0231bb16e83122a3044323ed13c323c9/lib/Linter.js#L41-L43

Comment (philippfromme) [reply lib/utils/error-messages.js:399]: Right now there is no use case for Camunda 7. But you're right, we could use the execution platform from the XML. So, in that case, we'd also assume that `allowedVersion` refers to the same execution platform. I'll adjust.

Comment (philippfromme) [reply lib/utils/error-messages.js]: Done: https://github.com/camunda/linting/pull/34/commits/afac378d99e232ad48bf9da59ba238da3a8f2a7c ✔️

**Review by marstamm — APPROVED** [review]
👏

**Pull request merged**